### PR TITLE
Update getBalanceOfMainChain to support ERC20

### PR DESCRIPTION
**Pull request by syuhei176**
```
const balance = await wallet.getBalanceOfMainChain(tokenId)
console.log(balance.toNumber()
```

### CODE REVIEW

File: packages/wallet/src/wallet.ts [hunk 1]
```diff
@@ -413,8 +414,18 @@ export class ChamberWallet extends EventEmitter {
...
   }
 
-  getBalanceOfMainChain() {
-    return this.wallet.getBalance()
+  async getBalanceOfMainChain(_tokenId?: number): Promise<BigNumber> {
+    const tokenId = _tokenId || 0
+    if(tokenId == 0) {
+      return this.wallet.getBalance()
+    } else {
+      // had better to use cache?
```
Comment (syuhei176) [inline packages/wallet/src/wallet.ts]: This code calls `ERC20.balanceOf` directly. Should I use cache?

File: packages/wallet/src/wallet.ts [hunk 2]
```diff
@@ -413,8 +414,18 @@ export class ChamberWallet extends EventEmitter {
     return this.wallet.address
   }
 
-  getBalanceOfMainChain() {
-    return this.wallet.getBalance()
+  async getBalanceOfMainChain(_tokenId?: number): Promise<BigNumber> {
```
Comment (syuhei176) [inline packages/wallet/src/wallet.ts]: I change `getBalanceOfMainChain` to async function. Is it OK?

Comment (syuhei176) [reply packages/wallet/src/wallet.ts]: > I change getBalanceOfMainChain to async function.

This is misunderstanding. original methods was async function.

File: packages/wallet/src/wallet.ts (hunk 1 above)
Comment (shogochiai) [reply packages/wallet/src/wallet.ts]: Design choice: cache handler on sdk, or on redux

**Pull request merged**